scio-test/src/test/scala/com/spotify/scio/testing/JobTestTest.scala

```diff
@@ -86,6 +87,27 @@ object PubsubJob {
   }
 }
 
+object PubsubInWithAttributesJob {
+  def main(cmdlineArgs: Array[String]): Unit = {
+    val (sc, args) = ContextAndArgs(cmdlineArgs)
+    sc.pubsubSubscriptionWithAttributes(args("input"))
+      .map((v: (String, Map[String, String])) => (v._1 + "X", v._2))
+      .saveAsPubsubWithAttributes(args("output"))
```

Comment:
This is to illustrate the use of the parameters.
I tried ```.map{ case  (value : String, attributes :  Map[String, String]) => (  value + "X", attributes ) }```
But it doesn't compile